packages/validator/src/validator.ts

```diff
@@ -19,12 +19,25 @@ import {toHexString} from "@chainsafe/ssz";
 import {ValidatorEventEmitter} from "./services/emitter";
 import {ValidatorStore, Signer} from "./services/validatorStore";
 import {computeEpochAtSlot, getCurrentSlot} from "@chainsafe/lodestar-beacon-state-transition";
+import {SecretKey} from "@chainsafe/bls";
+import {BlockDutiesService} from "./services/blockDuties";
+import {AttestationDutiesService} from "./services/attestationDuties";
+import {SyncCommitteeDutiesService} from "./services/syncCommitteeDuties";
+import {PubkeyHex} from "./types";
+
+export type SecretKeyInfo = {
+  secretKey: SecretKey;
+  keystorePath?: string;
+  unlockSecretKeys?: () => void;
```

Comment:
Does really the validator have to be aware that keystores require an unlock call? The keymanager should handle this on its end. Feels like leaked separation of concerns